docs/wallet/web-wallet.md

```diff
@@ -89,6 +89,10 @@ The Web Wallet can be seamlessly integrated into the credential issuance flow by
 
 You can experience a demo of the Web Wallet integration [here](https://web-wallet-demo.privado.id/).
 
+:::caution
+The prior method of integrating the Web Wallet using `https://wallet.privado.id/#base64EncodedData` is now deprecated and will no longer be supported after Release 9.
```

Comment:
Release 8 you mean?

Reply:
Nope release 9, we will support it for the time being.
Agreed with Sasha on this

Reply:
Ahh sorry I misread, you say we are removing support for the old format in the next release. Cool.